### PR TITLE
Add image ratio transform

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
cc: @andygnewman

### CODE REVIEW

File: lib/image-ratio-transform.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = mainImage => {
+	if (!mainImage) return;
+
+	return mainImage.width / mainImage.height;
```
Comment (andygnewman) [inline lib/image-ratio-transform.js]: Whilst most images have width and height properties, there may be some circumstances where they don't.

is the behaviour for undefined properties acceptable?

Comment (ironsidevsquincy) [inline lib/image-ratio-transform.js]: Ah, ok, I'll put in a check

**Comment by andygnewman** [comment]
I'll fix the test coverage when I put into the new version :) 👍

**Comment by ironsidevsquincy** [comment]
Ah, yeah... Thanks!

**Pull request merged**